Dockerfile

```diff
@@ -76,142 +79,102 @@ RUN wget https://github.com/samtools/htslib/releases/download/1.9/htslib-1.9.tar
 #### R packages
 ###############
 
-# Commonly used R packages
+# From CRAN and Bioconductor
 RUN ./install_bioc.r \
+    annotatr \
+    BSgenome.Hsapiens.UCSC.hg19 \
+    BSgenome.Hsapiens.UCSC.hg38  \
+    caret \
     class \
     cluster \
+    cmprsk \
+    ComplexHeatmap \
+    corrplot \
+    d3r \
     data.table \
+    deconstructSigs \
     DT \
+    e1071  \
+    EnsDb.Hsapiens.v86 \
+    ensembldb \
+    EnvStats \
     flextable \
     foreign \
+    gdalUtils \
+    GenVisR \
     GGally \
+    ggbio \
+    ggforce \
+    ggfortify \
+    ggpubr \
+    ggrepel \
+    ggsci \
+    ggsignif \
+    glmnet \
+    glmnetUtils \
+    gplots \
+    gridGraphics \
+    GSVA \
+    hexbin \
+    janitor \
     lattice \
+    leafem \
+    leafpop \
+    lsa \
+    lwgeom \
+    mapview \
     MASS \
     Matrix \
     mgcv \
+    MM2S \
```

Comment:
This might be one of the ones to _not_ group together. This is what's in master now - 
```
# Molecular subtyping MB
RUN R -e "remotes::install_github('d3b-center/medullo-classifier-package', ref = 'e3d12f64e2e4e00f5ea884f3353eb8c4b612abe8', dependencies = TRUE, upgrade = FALSE)" \
    && ./install_bioc.r MM2S
# More recent version of sva required for molecular subtyping MB
RUN R -e "remotes::install_github('jtleek/sva-devel@123be9b2b9fd7c7cd495fab7d7d901767964ce9e', dependencies = FALSE, upgrade = FALSE)"
```

Conceptually `MMS2` is coded there like `TCGAbiolinks` is coded here. 

Either way nothing is _missing_, so this is just an organizational aspect at your discretion.